Library/OptimizelyClientType.swift

```diff
@@ -32,3 +35,28 @@ extension OptimizelyClientType {
     return variant
   }
 }
+
+public func optimizelyTrackingAttributesAndEventTags(
+  with user: User?,
+  project: Project,
+  refTag: RefTag?
+) -> ([String: Any], [String: Any]) {
+  let properties: [String: Any] = [
+    "backings_count": user?.stats.backedProjectsCount,
+    "location": user?.location?.country.lowercased(),
+    "os_version": AppEnvironment.current.device.systemVersion,
+    "logged_in": user != nil,
+    "chosen_currency": project.stats.currentCurrency,
+    "locale": AppEnvironment.current.locale.identifier
+  ]
+  .compact()
+
+  let eventTags: [String: Any] = [
+    "project_subcategory": project.category.name,
```

Comment:
This _might_ not be the property that we want for this. The spreadsheet mentions "What project category has the user pledged previously?" - I've asked for confirmation.